assets/css/style.css

```diff
@@ -0,0 +1,708 @@
+}
+
+@font-face {
+  font-family: Cocogoose-Pro-Block-Innerline;
+  src: url(/first-capstone-project/assets/fonts/cocogoose-pro/Cocogoose-Pro-Block-Innerline.ttf);
+}
+
+@font-face {
+  font-family: Cocogoose-Pro-Light;
+  src: url(/first-capstone-project/assets/fonts/cocogoose-pro/Cocogoose-Pro-Light.ttf);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato-Regular', sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:visited {
+  text-decoration: none;
+  color: #ec5242;
+}
+
+body {
+  background-color: #f5f5f5;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+ul {
+  list-style-type: none;
+}
+
+footer img {
+  width: 11rem;
+}
+
+.navbar > img {
+  width: 24px;
+}
+
+.desktop {
+  display: none;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  background-image: url(/first-capstone-project/assets/images/bg2.png);
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
+
+.socials {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.socials:hover,
+.userlinks:hover {
+  cursor: pointer;
+}
+
+.hero {
+  display: flex;
+  opacity: 0.7;
+  flex-direction: column;
+  gap: 20px;
+  background-color: #f5f5f5;
+  margin-top: 4rem;
+}
+
+.hero img {
+  width: 100px;
+}
+
+.hero h1 {
+  text-align: start;
+  text-transform: uppercase;
+  font-family: 'Lato-Bold', sans-serif;
+  font-size: 2rem;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+  font-family: 'Lato', sans-serif;
+  color: #f81404;
+}
+
+.headline-about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 2rem;
+  font-family: 'Lato', sans-serif;
+  color: #f81404;
+}
+
+.headline > p {
+  font-size: 1.5rem;
+  margin-bottom: 1rem;
+}
+
+.headline-about > p {
+  font-size: 1.3rem;
+  margin-bottom: 1rem;
+}
+
+.headline h1 {
+  text-transform: uppercase;
+  color: #f81404;
+}
+
+.title-h1 {
+  background: url(/first-capstone-project/assets/images/title-bg.jpg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: repeat;
+  -webkit-background-clip: text;
+  /* stylelint-disable-next-line csstree/validator */
+  -webkit-text-fill-color: transparent;
+}
+
+.card {
+  border: 4px solid;
+  border-color: #fff;
+  border-radius: 5px;
+  padding: 10px 5px;
+  text-align: start;
+  margin: auto;
+  margin-left: 0;
+}
+
+.card-about {
+  border: 1px solid;
+  border-color: #cdc4c4;
+  background-color: #fff;
+  border-radius: 5px;
+  padding: 10px 5px;
+  text-align: center;
+  width: 80%;
+  margin: auto;
+}
+
+.card p {
+  font-family: 'Lato-Medium', sans-serif;
+  font-size: 18px;
+}
+
+q {
+  font-family: 'Lato-Italic', sans-serif;
+}
+
+.values {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(/first-capstone-project/assets/images/values-bg.png);
+
+  /* background-repeat: no-repeat; */
+
+  /* background-size: cover; */
```

Comment:
- [ OPTIONAL] Please make sure that you remove any commented code to keep your code clean and tidy :pray: